org.kde.CrowTranslate.yaml

```diff
@@ -0,0 +1,64 @@
+id: org.kde.CrowTranslate
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # The app supports capturing screen via KWin and Mutter DBus APIs, but
+  # access to it seems overly broad in spite of the app also supporting XDP API
+  #- --talk-name=org.kde.KWin
+  #- --talk-name=org.gnome.Shell
+rename-icon: crow-translate
+copy-icon: true
+command: crow
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+modules:
+  - name: crow-translate
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://invent.kde.org/office/crow-translate.git
+        tag: 2.11.1
+        commit: 7113ff01a994584ed0a3ec557fc3a694fcb06b63
+      - type: patch
+        paths:
+          - patches/crow-translate/metainfo.patch
+        x-checker-data:
+          type: git
+    modules:
+      - name: tesseract
+        buildsystem: cmake-ninja
+        builddir: true
+        config-opts:
+          - -DBUILD_SHARED_LIBS:BOOL=ON
+        cleanup:
+          - /bin
+        sources:
```

Comment:
You should add `x-checker-data` config for all sources

Reply:
I checked other projects and looks like they don't use `x-checker-data` for archives from GitHub.
For example, [org.flathub.flatpak-external-data-checker](https://github.com/flathub/org.flathub.flatpak-external-data-checker/blob/502b06d4d4e67372ee1c699be27d920995c70e90/org.flathub.flatpak-external-data-checker.yml#L159C27-L159C34) does not do this.